packages/perl-debug-adapter/package.yaml

```
@@ -0,0 +1,20 @@
+---
+name: perl-debug-adapter
+description: Perl debugger
+homepage: https://github.com/Nihilus118/vscode-perl-debug
+licenses:
+  - MIT
+languages:
+  - Perl
+categories:
+  - DAP
+
+source:
+  id: pkg:github/Nihilus118/vscode-perl-debug@7ef298610b91b76ed76bebc4962f1a849052a437
```

It'd be ideal if they tagged releases, seeing as they do version it. Even better would be if they packaged this as a npm package. Feel free to ask them about this ;)

```suggestion
  # renovate:datasource=git-refs
  id: pkg:github/Nihilus118/vscode-perl-debug@7ef298610b91b76ed76bebc4962f1a849052a437
```